src/app/modules/customer-management/services/customer.service.ts

```diff
@@ -24,4 +24,17 @@ export class CustomerService {
   getCustomers(): Observable<any> {
     return this.http.get(this.baseUrl);
   }
+
+  deleteCustomer(customerId: string): Observable<any> {
+    const url = `${this.baseUrl}/${customerId}`;
+    return this.http.delete(url);
+  }
+
+  updateCustomer(customerData): Observable<any> {
+    const url = `${this.baseUrl}/${customerData.id}`;
+    const body = {
+      ...customerData,
+    };
```

Comment:
body object it's not necessary, please remove it and send directly customerData. 
Please remove it also remove in createCustomer method, and remove the tenantId.

Reply:
done